modern-gpus/intel-gpu.md

```diff
@@ -11,10 +11,10 @@ More info:
 ## Native Intel iGPUs
 
 ### **Ivy Bridge 3XXX**
-#### Highest Supported OS: Catalina(10.15)
-#### Initial Supported OS: Lion(10.7)
+#### Highest Supported OS: Catalina (10.15)
+#### Initial Supported OS: Lion (10.7)
 
-Regarding the HD 4000, it's completely native with Catalina(Though no longer supported with macOS 11: Big Sur). The HD 2500 on the other hand only has partial support in Mojave for quick sync features as hardware acceleration is [unsupported](https://olarila.com/forum/viewtopic.php?t=7714) so you will need a compatible dGPU for display purposes. Initial supported introduced with macOS 10.8
+Regarding the HD 4000, it's completely native with Catalina (though no longer supported with macOS Big Sur). The HD 2500, on the other hand, only has partial support in Mojave for quick sync features, but hardware acceleration is unsupported, so you will need a compatible dGPU for display purposes.
 
 * Note: As of macOS 11.3, the HD 4000 drivers are still present in Big Sur. Please keep in mind future OS updates and releases may remove these drivers.
```

Comment:
Note not needed anymore

Reply:
> Note not needed anymore

Confused about what you'd like me to do :P

Reply:
```
* Note: As of macOS 11.3, the HD 4000 drivers are still present in Big Sur. Plase keep in mind....
```
They are in all versions of Big Sur, and were dropped in Monterey.
Probably need to update the Highest Support OS to Big Sur as well.

They weren't officially supported by Apple but continued to work 100% fine assuming you used a newer SMBIOS. Please don't use MacPro6,1 for a computer using the iGPU.

Reply:
Gotcha. Forgot about Mac Pros not using iGPUs, I'll update that.

Reply:
Resolved with 3acbabc